projects/flimnap/main.cc

```diff
@@ -66,10 +66,10 @@ namespace {
     constexpr float fixed_velocity = 5.0f;
 
     // transmission
-    std::array<uint8_t, SimulationMessage_size> encode_buffer;
-    std::array<uint8_t, cobs::max_encoded_length(SimulationMessage_size)> packet_buffer;
+    constexpr std::size_t VARINT_MAX_SIZE = 10;
```

Comment:
Only relevant definition in nanopb is this function: https://github.com/nanopb/nanopb/blob/651bdc45f2180b17c132470ff1a3a515dbffaa78/generator/nanopb_generator.py#L124-L136

Could not find a definition in google's protobuf implementation (https://github.com/google/protobuf/search?l=C%2B%2B&q=varint&type=&utf8=%E2%9C%93), except for this one in a test: https://github.com/google/protobuf/blob/ffa71f80070fde3eb8e6a5779df951c4fe0729e0/conformance/conformance_test.cc

The fact that its defined in this test suggests there is no library definition of the maximum varint byte size.